test/ui/DatePickerTest.ts

```diff
@@ -21,6 +21,10 @@ export function DatePickerTest() {
       expect(element.readOnly).toBe(true);
     });
 
```

Comment:
Some missing tests :slightly_smiling_face: 

- `getDateValue`
- `this.onChange(this);` is now in `reset()`